config/locales/en.yml

```diff
@@ -28,27 +28,33 @@ en:
   name: 'Name'
   save: 'Save'
   saving: 'Saving'
-  
+
   drive_the_vote: 'Drive the Vote'
   home: 'Home'
   faq: 'faq'
   about: 'About'
-  coming_soon: 'Coming soon'
   click_here: 'Click here'
   to_be_notified: 'to be notified'
   learn_more: 'Learn more'
   contact: 'Contact'
   your_profile: 'Your profile'
-  your_settings: 'Your settings'  
+  your_settings: 'Your settings'
   short_description: |
-    arranges free rides
-    to the polls on election day.
-  # home2: 'You can sign up as either a driver or a passenger--or just join the community to get information about where you polling place is, who your elected officials are, and whose running for office in your community.'
-  election_is_on: 'The 2016 Election is on'
-  election_date: 'Tuesday, November 8'
-  thats: "That's"
-  days_away: 'days from now'
+    free rides to and from the polls on election day.
+  how_it_works: 'How it Works'
+  how_it_works_copy: |
+    Voters send a text to request a ride, and Drive the Vote connects them with a volunteer driver, who picks them up and takes them to their polling place.
+  how_to_help: 'How to Help'
+  how_to_help_copy: |
+    If you have a car and can be in <b>Florida</b>, <b>Ohio</b>, <b>Pennsylvania</b>, <b>North Carolina</b>, <b>Michigan</b>, or <b>Georgia</b> on election day, volunteer to be a driver! Help get voters to the polls on November 8th and make sure everyone has a vote.
```

Comment:
That would require having dispatchers on duty... from now until the election, I think. Let's think on how we might be able to support that, but right now I don't see how we can.